site/src/docs/components/cookie-consent/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+slug: '/components/cookie-consent'
+title: 'CookieConsent'
+navTitle: 'CookieConsent'
+---
+
+import { Link } from 'hds-react';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+This documentation page is about HDS CookieCompliance component. If you are looking for documentation about cookies in general, please refer to <a href="/patterns/cookies">HDS cookie pattern documentation page</a>.
+
+### Example
+
+<Link href="/storybook/react/?path=/story/components-cookieconsent--modal-version">View modal example in Storybook</Link>
```

Comment:
Same problem here, the Link is not working on the demo site. We need to use the gatsby `withPrefix`. Also because the link is to a storybook, that is outside the current docsite app, we should probably add the prop `openInNewTab` to open the link in to a new tab

Reply:
Fixed